### PR TITLE
LPS-188040 Guest user can access non-published Web Content

### PR DESCRIPTION
**Pull request by Mikellorza**
Fix bug : https://liferay.atlassian.net/browse/LPS-188040

Description
-------
We have a bug when we are showing a web content in a pending status to the guest users. So, this PR fix the bug hiding the content if the user is not logged and the status of the model is in a pending status.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Mikellorza** [comment]
ci:test:sf

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/layout/display/page/JournalArticleLayoutDisplayPageProvider.java [hunk 1]
```diff
@@ -92,8 +94,13 @@ public String getClassName() {
 		try {
 			JournalArticle article = _getArticle(groupId, urlTitle, version);
 
+			ServiceContext serviceContext =
+				ServiceContextThreadLocal.getServiceContext();
+
 			if ((article == null) || article.isExpired() ||
-				article.isInTrash()) {
+				article.isInTrash() ||
```
Comment (ealonso) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/layout/display/page/JournalArticleLayoutDisplayPageProvider.java:101]: Hey @Mikellorza, I think we should apply this fix to the other methods, also the isExpired check

Comment (Mikellorza) [reply modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/layout/display/page/JournalArticleLayoutDisplayPageProvider.java:101]: Ok @ealonso , I am going to check it

Thanks

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**